sections/semantics-scriptings.include

```diff
@@ -2456,14 +2456,87 @@
     </dd>
   </dl>
 
-  The <{slot}> element creates a named <a>slot</a> within a <a>shadow tree</a>.
-  If it has an <a>assigned node</a> it <a>represents</a> that, if not it <a>represents</a> its contents.
+  The <{slot}> element creates a named <a>slot</a> within a <a>shadow tree</a>. If it has an
+  <a>assigned node</a> it <a>represents</a> that, if not it <a>represents</a> its contents.
 
   The <dfn element-attr for="slot">name</dfn> attribute's value is a string.
   It <a>represents</a> the name of the <a>slot</a> created in a shadow tree.
 
   The {{slot/name}} IDL attribute must <a>reflect</a> the <{slot/name}> content attribute.
 
+  <div class="example">
+    The following example shows how the <{slot}> element may be used in constructing a reusable
```

Comment:
I think we could phrase this a bit more simply. I'll make a PR on this branch...